src/test/resources/goal-output/commons-math3-3.6.1/org.apache.commons.math3.complex.Complex_goal.json

```diff
@@ -429,7 +429,7 @@
           "this"
         ],
         "comment": "if this is zero",
-        "condition": "target.abs()==0",
+        "condition": "target==0",
```

Comment:
You are right. I'm going to restore the goal file. 
By the way, since the translation is wrong, I think that the type compatibility check of `this` is missing.